aspnetcore-features/GettingPropertyMappingsWithAutomapper/Test/GettingPropertyMappingsWithAutomapperTest.cs

```diff
@@ -0,0 +1,12 @@
+namespace Test
+{
+    [TestClass]
+    public class GettingPropertyMappingsWithAutomapperTest
+    {
+        [TestMethod]
+        public void WhenProgrammeExecutionThenCorrectOutputLiveTest()
+        {
+            
```

Comment:
There should be at least one test in here. It seems like you could re-use some of the same code you're running in the `Program` class, and verify that the mapping is working correctly.